lib/leaderboard/leaderboard_client.ts

```diff
@@ -121,3 +121,5 @@ export interface LeaderboardClient {
    */
   listSeasons(): Promise<Season[]>;
 }
+
+// TODO: Calculate the top scorers of a given season to eventually be formatted in a leaderboard table.
```

Comment:
Resolve TODO comment.

Reply:
Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/33/commits/186abd9c5005d861ae188bef586828a908986cb9>.